webrender/src/debug_render.rs

```diff
@@ -416,8 +422,11 @@ impl DebugRenderer {
 
             // Glyph
             if !self.font_indices.is_empty() {
+                #[cfg(feature = "gl")]
                 device.bind_program(&self.font_program);
                 device.set_uniforms(&self.font_program, &projection);
+                #[cfg(not(feature = "gl"))]
+                device.bind_program(&self.font_program);
```

Comment:
starting to suspect it's intentional, but why?

Reply:
@kvark this turned out to be my silliness. Removed the duplication since we don't need it.